packages/playground/blueprints/src/lib/compile.ts

```diff
@@ -86,6 +87,13 @@ export function compileBlueprint(
 		...blueprint,
 		steps: (blueprint.steps || []).filter(isStepDefinition),
 	};
+	// Convert legacy importFile steps to importWxr
+	for (const step of blueprint.steps!) {
+		if (typeof step === 'object' && step!.step === 'importFile') {
+			(step as any).step = 'importWxr';
+		}
+	}
```

Comment:
thanks for adding a smooth transition for old code.
should we log something indicating that the step has a new name?

Reply:
Good call! I just added it in [d731248](https://github.com/WordPress/wordpress-playground/pull/1192/commits/d73124866bd15580e77188b2332f64e64c2afac5)